src/Hashing/Argon2IdHashing.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+/**
+ * This file is part of the Zest Framework.
+ *
+ * @author   Muhammad Umer Farooq <lablnet01@gmail.com>
+ * @author-profile https://www.facebook.com/Muhammadumerfarooq01/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @since 3.0.0
+ *
+ * @license MIT
+ */
+
+namespace Zest\Hashing;
+
+class Argon2IdHashing extends AbstractHashing
```

Comment:
This class is separated from `ArgonHashing` class because the original `ArgonHashing` class do two different algorithms.

They should be separated.